cmd/harmony/main.go

```diff
@@ -371,6 +365,36 @@ func setupConsensusKey(nodeConfig *nodeconfig.ConfigType) multibls.PublicKey {
 	return *consensusMultiPubKey
 }
 
+func setupHost(nodeConfig *nodeconfig.ConfigType) (p2p.Host, error) {
+	baseDS := datastore.NewMapDatastore()
+	const DevRendezVousPoint = "/ip4/167.99.223.55/tcp/4040/p2p/QmTo3RS6Uc8aCS5Cxx8EBHkNCe4C7vKRanbMEboxkA92Cn"
```

Comment:
Dev? Is this  the bootnode? Do we need to change bootnode setup for the network?

Reply:
no - there is no more bootnode as separate project binary. we link up using IPFS public nodes + a seed node of ours, this is a leftover seednode I was using - will update